_config.yml

```diff
@@ -1,11 +1,25 @@
 title: Aquarium by Aquarist Labs
 
-description: >-
-  Experimenting on captive cephalopods since 2021
+description: >
+  This work is licensed under a Creative Commons
```

Comment:
Why this change? This is the short summary of the site, tagline etc; is that the best place for the license statement?

Reply:
Ah, because I changed the footer. I'll show you the output

Reply:
![Screenshot from 2021-04-21 14-11-30](https://user-images.githubusercontent.com/1987994/115559288-7de1f600-a2ab-11eb-926b-82eba528657e.png)

Now it's more inline with other website styling.